src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -37,12 +37,13 @@
     DataPointT,
     NumberDataPoint,
     Metric,
-    HistogramDataPoint
+    HistogramDataPoint,
+    PeriodicExportingMetricReader
 )
 
 VERSION = "0.3.0-rc1"
 
-DYNATRACE_TEMPORALITY_PREFERENCE = {
+_DYNATRACE_TEMPORALITY_PREFERENCE = {
```

Comment:
Yes it is. I also did not see any.

Reply:
> Looks good to me! I have only one suggestion: The methods order inside `__init__.py` are a bit hard to follow/read. It would be nicer I think if they were positioned in the order they are called. E.g.
> 
>     1. `export`
> 
>     2. `_to_dynatrace_metric`
>        2.1 `_monotonic_sum_to_dynatrace_metric`
>        ...
> 
>     3. `_histogram_to_dynatrace_metric`
>        ...
> 
> 
> 4 `_send_lines`
> 
> Then the "utilities" one can be at the very bottom, like `_log_temporality_mismatch`
> 
> Also the ones on the very top `_get_histogram_max` can also be marked as static and placed at the bottom as well (or in a separated file, if you prefer).

I previously held off on moving things to another file as I thought it would blow up the diff too much. Since now two reviewers asked for it, I split this up into `exporter`, `histogram_utils` and `factory` in [82e7561](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/82e756162b8a90cbe1e254e2ce99afa57ad7cc2a). I also changed the ordering in [3d0c974](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/3d0c974f7d8496dcab142d516d92238442d13c4b).